vcd/resource_vcd_vapp.go

```diff
@@ -485,6 +540,26 @@ func resourceVcdVAppRead(d *schema.ResourceData, meta interface{}) error {
 	if err != nil {
 		return fmt.Errorf("unable to set guest properties in state: %s", err)
 	}
+	// _ = d.Set("power_on", vapp.VApp.Status == 4)
```

Comment:
Is this missing?

Reply:
This field works well as input, but not as output. The status of "POWERED_OFF" corresponds to VApp.Status == 4, but this is not always true when we check in our current tests. The vApp will be eventually set to that state, but not immediately. That's why I added `status` and `status_text`, so that users have some visibility on what the vApp is up to.